test/puppetlabs/puppetdb/anonymizer_test.clj

```diff
@@ -162,7 +162,9 @@
 (deftest test-anonymize-leaf-message
   (testing "should return a string of equal length"
     (is (string? (anonymize-leaf-memoize :message "good old string")))
-    (is (= 15 (count (anonymize-leaf-memoize :message "good old string"))))))
+    (is (= 15 (count (anonymize-leaf-memoize :message "good old string"))))
+    (is (string? (anonymize-leaf-memoize :text "good old string")))
+    (is (= 15 (count (anonymize-leaf-memoize :text "good old string"))))))
```

Comment:
Think I'd go ahead and verify that we get exactly what we expect here, i.e. "???????????????".